packages/builders/src/jest/schema.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
```

Comment:
Some description are title case and some note. Please take a look at how angular builders descriptions are written, and use consistent style.